source/reference/configure-shell-settings-global.txt

```diff
@@ -0,0 +1,104 @@
+.. _configure-settings-global:
+
+===========================================================
+Configure :binary:`~bin.mongosh` Using a Configuration File
+===========================================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+
+You can specify ``mongosh`` settings in a global configuration file.
+When you specify settings in a configuration file, those settings are
+applied at startup.
```

Comment:
[question/thought]

Is it worth mentioning here that changes won't take places until folks restart the server?